packages/core/tests/controllers/ApiController.test.ts

```diff
@@ -10,7 +10,8 @@ describe('ApiController', () => {
       featured: [],
       recommended: [],
       wallets: [],
-      search: []
+      search: [],
+      isAnalyticsEnabled: false
```

Comment:
yay! isn't this a breaking change?

Reply:
this was already deployed, but the test was missing. It's not breaking cause the default is false.

Reply:
ah my bad, thought it was isActivityEnabled